test/cql-pytest/test_describe.py

```diff
@@ -2567,3 +2568,90 @@ def aux(row: DescRow) -> DescRow:
             assert restore_stmts == res
         finally:
             cql.execute(f"DROP KEYSPACE IF EXISTS {ks}")
+
+### ===========================================================================
```

Comment:
I guess I was sleeping under a rock when you wrote most of test/cql-pytest/test_describe.py (almost 2500 lines of code) and included in comments not just helpful text but also all those lines full of equal signs, hash signs, and dots. I find all those lines pointless, but maybe I'm too late.